quinn/examples/connection.rs

```diff
@@ -0,0 +1,62 @@
+//! This example intends to use the smallest amout of code to make a simple QUIC connection.
+//!
+//! The server issues it's own certificate and passes it to the client to trust.
+//!
+//! Run:
+//! ```
+//! $ cargo run --example connection
+//! ```
```

Comment:
I think you need to annotate all these code blocks (probably as text) so rustdoc doesn't try to compile and run them as doctests. Might not be an issue for examples in practice, but good style to be explicit all the same.

Reply:
Makes sense, will change it. Although  `cargo doc` builds successfully.